RockLib.UniversalMemberAccessor.Tests/TypeCreator/Define.cs

```diff
@@ -59,27 +59,27 @@ public static ConstructorDefinition Constructor(Visibility visibility = Visibili
                     }
                 }
 
-                il.Emit(OpCodes.Ldstr, tb.Name + ".ctor(" + string.Join(", ", parameters.Select(p => p.Type.Name)) + ")");
-                il.Emit(OpCodes.Call, typeof(Console).GetMethod("WriteLine", new[] { typeof(string) }));
+                il.Emit(OpCodes.Ldstr, tb.Name + ".ctor(" + string.Join(", ", parameters.Select(p => p.Type?.Name)) + ")");
```

Comment:
I'd consider changing this to string interpolation. Especially with perf improvements around string interpolation in C# 10.

Reply:
Updated to string interpolation.